apps/ex_wire/config/dev.exs

```diff
@@ -2,7 +2,10 @@ use Mix.Config
 
 config :ex_wire,
   network_adapter: {ExWire.Adapter.UDP, NetworkClient},
-  sync: false,
+  sync: true,
```

Comment:
I also left the `sync` option to true since it seems like a more sensible default in my mind, but I can set that back to `false` if that makes more sense to people.